jekyll/_cci2/workspaces.md

```diff
@@ -88,17 +193,10 @@ For additional conceptual information on using workspaces, caching, and artifact
 
 Workspaces are stored for up to 15 days. Workspaces are not shared between pipelines, and the only way to access a workspace once the workflow has completed is if the workflow is rerun within the 15 day window.
 
-## Workspaces and runner network charges
-{: #workspaces-and-runner-network-charges }
-
-When using self-hosted runners there is a network and storage usage limit included in your plan. Once your usage exceeds your limit charges will apply. These charges are based on your accrued overages.
-
-For information on viewing your network and stoarage usage, and calculating your monthly network and storage overage costs, see the [Persisting Data]({{site.baseurl}}/2.0/persist-data/#managing-network-and-storage-use) guide.
-
```

Comment:
Just wondering why this section is to be removed?

Reply:
Hmm, I'm not actually sure... I don't remember getting rid of this 😬

Reply:
Okay I changed the wording slightly, but it's back in there. I think what might have happened is I cut it to move it to a different place and then never pasted it back in.